Integreat/Integreat.Shared/ViewModels/BaseWebViewViewModel.cs

```diff
@@ -51,7 +52,19 @@ public async Task OnNavigating(WebNavigatingEventArgs eventArgs)
                 && (eventArgs.Url.ToLower().Contains(".pdf")
                     || eventArgs.Url.ToLower().Contains("_pdf")))
             {
-                await ShowPdfPage(eventArgs);
+                if (eventArgs.Url.ToLower().EndsWith(".tmpextensions"))
+                {
+
+                    var pdfLocalPath = WebUtility.UrlDecode(eventArgs.Url.ToLower());
+                    var regex = new Regex(Regex.Escape("http:/"));
```

Comment:
The idea here is that the path changes from http://some/local/path/file.pdf to /some/local/path/file.pdf?

Reply:
The issue ist that Android not notice .pdf as link so I wrap it with some stuff so android believe it is a link...

And after the link was triggered I remove the stuff which I added...

But a problem is I did this in the background downloader so we have to wait until this file was loaded... Another approach here could be to do this than the page will be loaded so we can be sure the the target ends not with .pdf